libraries/entities/src/DiffTraversal.cpp

```diff
@@ -166,7 +192,12 @@ DiffTraversal::Type DiffTraversal::prepareNewTraversal(
     //
 
     Type type;
-    if (_completedView.startTime == 0) {
+    if (!usesFrustum) {
+        type = Type::FullScene;
+        _getNextVisibleElementCallback = [&](DiffTraversal::VisibleElement& next) {
+            _path.back().getNextVisibleElementFullScene(next, _completedView.startTime);
+        };
+    } else if (_completedView.startTime == 0) {
         type = Type::First;
         _currentView.viewFrustum = viewFrustum;
         _currentView.lodLevelOffset = root->getLevel() + lodLevelOffset - 1; // -1 because true root has level=1
```

Comment:
Should `if (nextIndex > 0)` on line 242 below this be `if (nextIndex >= 0)`?

Reply:
Never mind, the answer to that question is no because the _getNextVisibleElementCallback increments _nextIndex, so nextIndex will actually equal 0 for the root.